src/core/core/global_config.cpp

```diff
@@ -443,11 +468,36 @@ bool service_spec::init_app_specs()
         ))
         return false;
 
-    std::vector<std::string> allSectionNames;
-    config->get_all_sections(allSectionNames);
+    std::vector<std::string> all_section_names;
+    config->get_all_sections(all_section_names);
     
+    // check mimic app
+    if (enable_default_app_mimic)
+    {
+        std::string mimic_section_name("apps.mimic");
+        if (std::find(all_section_names.begin(), all_section_names.end(), mimic_section_name)
+            == all_section_names.end())
+        {
```

Comment:
maybe we can auto create default [apps.mimic] section if this section is not exist in config:
```
[apps.mimic]
type = dsn.app.mimic
count = 1
run = true
pools = THREAD_POOL_DEFAULT
```
It will be more convenient to use, when user only need to set "enable_default_app_mimic".